core/src/test/java/apoc/trigger/TriggerClusterRoutingTest.java

```diff
@@ -0,0 +1,79 @@
+package apoc.trigger;
+
+import apoc.util.Neo4jContainerExtension;
+import apoc.util.TestContainerUtil;
+import apoc.util.TestcontainersCausalCluster;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.neo4j.driver.Session;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static apoc.trigger.Trigger.SYS_NON_LEADER_ERROR;
+import static apoc.trigger.TriggerNewProcedures.TRIGGER_NOT_ROUTED_ERROR;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class TriggerClusterRoutingTest {
+
+    private static TestcontainersCausalCluster cluster;
+
+    @BeforeClass
+    public static void setupCluster() {
+        cluster = TestContainerUtil
+                .createEnterpriseCluster(3, 1, Collections.emptyMap(), Map.of(
+                        "NEO4J_dbms_routing_enabled", "true",
+                        "apoc.trigger.enabled", "true"
+                ));
+    }
+
+    @AfterClass
+    public static void bringDownCluster() {
+        if (cluster != null) {
+            cluster.close();
+        }
+    }
+
+    @Test
+    public void testTriggerAddAndInstallAllowedOnlyInLeaderMember() {
```

Comment:
It's perhaps a small detail but could we split this test into two separate tests, please? 

Tests should probably only test one thing at a time to keep them simple. I would recommend one test for apoc.trigger.add and another separate test for apoc.trigger.install.

Reply:
Done